### PR TITLE
Add jitpack support for publishing

### PR DESCRIPTION
**Pull request by crocsandcoffee**
* Update README.md with new instructions
* Update project and build gradle with jitpack changes

Notes:
* I  checked and was able to successfully run the three apps (quick-start, sample and test) defined in this project.
* Here is what the README looks like on this branch: https://github.com/CanHub/Android-Image-Cropper/blob/jitpack/README.md
* Once this PR is merged, we create a new [GitHub Release](https://github.com/CanHub/Android-Image-Cropper/releases) with tag `1.1.1`, and JitPack will create a new artifact for this version.

### CODE REVIEW

File: cropper/build.gradle [hunk 1]
```diff
@@ -9,7 +12,7 @@ android {
         minSdkVersion 14
         targetSdkVersion rootProject.targetSdkVersion
         versionCode 1
-        versionName "1.1.0"
+        versionName "1.11.0"
```
Comment (crocsandcoffee) [inline cropper/build.gradle:15]: NOTE: Jitpack will build using GitHub releases + tag. The latest version is what it shown in the badge in README. It looks like a `1.10` tag is created  (https://github.com/CanHub/Android-Image-Cropper/releases). So the `versionName` will need to be higher I believe.

Here's what I am using for reference: https://jitpack.io/docs/

**Comment by crocsandcoffee** [comment]
I also checked and was able to successfully run the three apps (quick-start, sample and test) defined in this project.

File: cropper/build.gradle (hunk 1 above)
Comment (Canato) [reply cropper/build.gradle:15]: This tag is wrong. `1.10` was a mistype from `1.1.0` 

Let's change to `1.1.1` and I will delete the wrong one.

File: README.md [hunk 2]
```diff
@@ -13,60 +15,39 @@ Android Image Cropper
 
 [See GitHub Wiki for more info.](https://github.com/CanHub/Android-Image-Cropper/wiki)
 
+#### Step 1. Add the JitPack repository to your root build.gradle
 
-**The library is release with github Packages, if you already have github packages libraris can jump to second step**
-### 1. Generate a Personal Access Token for GitHub
-Reference: [Github Personal Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
```
Comment (Canato) [inline README.md:19]: Should we keep both? with the GitPackages ? 

If not, how this could affect who is already using gitPackages?

Comment (Canato) [reply README.md:19]: If we decide to not keep, please delete `publish.gradle` file

File: cropper/build.gradle [hunk 3]
```diff
@@ -1,6 +1,9 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 //apply from: file('publish.gradle')
```
Comment (Canato) [inline cropper/build.gradle:3]: If we will not keep GithubPackages, please delete this

**Review by Canato — CHANGES_REQUESTED** [review]
Sorry the long time to review this.

Christmas + Holidays

File: README.md (hunk 2 above)
Comment (Canato) [reply README.md:19]: I thought better, let's remove the Github Package way if we don't spot any issue for who is using it

Comment (crocsandcoffee) [reply README.md:19]: I believe it's fine for anyone who currently was using gitPackages. They would just need to add the jitpack repo to their project gradle and update the dependency and all should be fine. I will delete the `publish.gradle` next commit 👍🏻

File: cropper/build.gradle (hunk 3 above)
Comment (crocsandcoffee) [reply cropper/build.gradle:3]: 👍🏻 , will remove next commit

File: cropper/build.gradle (hunk 1 above)
Comment (crocsandcoffee) [reply cropper/build.gradle:15]: Fixed

**Review by Canato — APPROVED** [review]
Great work, tomorrow we release 1.1.1

Thanks for helping the lib :)

**Pull request merged**